### PR TITLE
React Final Capstone project

### PR DESCRIPTION
**Pull request by blase147**
In this PR,

I styled my components to match the design provided.
Deployed the project and tested for final details.
Recorded a video for my project.
Created a descriptive README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,33 +1,104 @@
...
+- **Fetches Covid-19 prevalence  data by continent**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+# Live-Demo
+> Here is a link (https://covid-19-updater-app.netlify.app/) to the live page.
+
+# Recorded video
+> Here is a link (https://www.loom.com/share/7936fa0318024f3dab367145e34598a3) to the recorded video.
+
```
Comment (Nemwel-Boniface) [inline README.md]: - [ ] Good job making the video as requested by the [previous reviewer](https://github.com/blase147/react-capstone/pull/1#discussion_r1041911526) However, you have not seen you showcase a description of the project as well as showcase an interesting piece of code. Kindly retake the video and resubmit it for review.

image

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
-        </a>
-      </header>
-    </div>
+    <Router>
+      <div>
+        <Heading />
+        <div>
+          <Routes>
+            <Route path="/" element={<Home />} />
+            <Route path="/details" element={<Details />} />
```
Comment (Nemwel-Boniface) [inline src/App.js]: - [ ] As requested by the previous reviewer, All your category routes have the same details as their route, but EACH item is required to have its own unique route. Then you can fetch the item using the unique identifier.

You should have something like this:

```path="details/:id"```

or

```path=":id"```

Suggestion: You can consider looking at the useParams from the react-router-dom package.

image

File: README.md [hunk 3]
```diff
@@ -39,32 +110,51 @@ Instead, it will copy all the configuration files and the transitive dependencie
...
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
 
-### Making a Progressive Web App
+## ⭐️ Show your support <a name="support"></a>
+Give a ⭐️ if you like this project!
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+## 🙏 🙏 Acknowledgments <a name="acknowledgements"></a>
+- I would like to thank [@microverseinc](https://github.com/microverseinc) for the templates and insights
```
Comment (Nemwel-Boniface) [inline README.md]: - [ ] As requested by the previous reviewer  Kindly add acknowledgment giving Nelson Sakwa proper credit for the design based on the Creative Commons license.

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @blase147     :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

Great job addressing some of the changes that were requested by a previous reviewer :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

- [ ] KIndly do not close/ open a new PR unless suggested by a code reviewer

- [ ] As requested by the [previous reviewer](https://github.com/blase147/react-capstone/pull/1#discussion_r1041927471), you were required to make sure that the tests are passing and not delete them altogether. Kindly add the tests back this time making sure that they pass.

image


_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (blase147) [reply src/App.js]: @Nemwel-Boniface Thanks for reviewing my code. However, I am still very much confused about exactly I need to go about this...seems like I need to refactor everything in my Details component? Please can you be a little more elaborate...

File: README.md (hunk 3 above)
Comment (peterrobert) [reply README.md]: This change has been done well

File: README.md (hunk 1 above)
Comment (peterrobert) [reply README.md]: This change has been done well.

File: src/App.js (hunk 2 above)
Comment (peterrobert) [reply src/App.js]: This change has been done well

**Review by peterrobert — APPROVED** [review]
Hi @blase147, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**